tests/test_api.py

```diff
@@ -465,11 +463,17 @@ def test_metadata_root(self):
         root.signed.add_key('root', key_metadata)
         self.assertEqual(pre_add_keyid, root.signed.roles['root'].keyids)
 
-        # Remove the key
-        root.signed.remove_key('root', keyid)
+        # Add the same key to targets as well
+        root.signed.add_key('targets', key_metadata)
 
-        # Assert that root does not contain the new key anymore
+        # Remove the key from root
+        root.signed.remove_key('root', keyid)
         self.assertNotIn(keyid, root.signed.roles['root'].keyids)
+        self.assertIn(keyid, root.signed.keys)
```

Comment:
Maybe add one clarifying comment before the `self.assertIn(...)` like:
`Make sure the key is still in keys dict as it's still used by targets`
Just to make it 100 % clear what you are checking here.

Reply:
@MVrachev, why do we need that clarification? What might confuse folks in that assert?

Reply:
Because he doesn't state what is the testing.
In his commit message, he explains that, but without that context is not immediately clear.
You are removing a key and still expect it to be in `root.signed.keys` why?